### PR TITLE
compile on Maui update

### PR DESCRIPTION
**Pull request by mand35**
minor changes

### CODE REVIEW

File: _lessons/maui-and-mahuika/12-building-code-on-maui.md [hunk 1]
```diff
@@ -135,7 +139,7 @@ SLURM_PARTITION=Debug mpiexec -n 6 simpleMPI
 
 Although the compiler drivers ```ftn```, ```cc``` and ```CC``` have a few options of their own, such as the ```-craype-verbose``` flag, they will pass through any additional compiler options to the underlying compiler. This means that you will still need to choose compiler flags that are specific to the Cray, Intel, or GNU compilers, and you will need to change them if you decide to switch compilers.
 
-For example, if you wanted to use the gfortran compiler, activate compiler warnings (```-Wall```), and require aggressive compiler optimisation (```-O3```), you would use the following commands:
+For example, if you wanted to use the gfortran compiler, activate compiler warnings (```-Wall```), and require aggressive compiler optimization (```-O3```), you would use the following commands:
```
Comment (chrisdjscott) [inline _lessons/maui-and-mahuika/12-building-code-on-maui.md]: @mand35 the spelling here was correct before (i.e. with the 's' not 'z') - could you change back (and same for the others below)?

**Review by mand35 — COMMENTED** [review]
changed back to BE

**Comment by tinyendian** [comment]
Just a comment on the "-homp" flag in the Cray compiler, this is not always the default according to the man page:

"If -O0 is implied or specified, then -h noomp is implied."

The reason why I added this section was that sometimes a dependent library contains OpenMP code while the main program doesn't, which can lead to linker errors due to missing OpenMP runtime libraries. So you have to make sure that "-homp" is set (explicitly or implicitly) at link time (or link the runtime libs yourself).

**Comment by mand35** [comment]
Good point. But on the other hand we also should point out that in most cases using the Cray compiler OMP is enabled by default. If one does not know and intend to compile the code without OMP support he/she will be surprised. 
Furthermore, -O0 is not default. So I don't see any conflict here.

**Comment by tinyendian** [comment]
Fair enough...

**Pull request merged**